components/org.wso2.carbon.identity.sso.saml/src/main/java/org/wso2/carbon/identity/sso/saml/validators/SPInitSSOAuthnRequestValidator.java

```diff
@@ -116,18 +107,31 @@ public SAMLSSOReqValidationResponseDTO validate() throws IdentityException {
                 return validationResponse;
             }
 
+            String issuerName = splitAppendedTenantDomain(issuer.getValue());
```

Comment:
shall we add a comment here what this do?

Reply:
https://github.com/wso2-extensions/identity-inbound-auth-saml/pull/290